config/betamocks/services_config.yml

```diff
@@ -901,6 +900,13 @@
       :cache_multiple_responses:
         :uid_location: "url"
         :uid_locator: "\/services/claims/v2/veterans\/(?:.+)\/claims\/(.+)/5103"
+    # ClaimsAPI BD
+    - :method: :post
+      :path: "/services/benefits-documents/v1/documents"
+      :file_path: "/lighthouse/claims_api/benefits_documents/upload"
+    - :method: :post
+      :path: "/services/benefits-documents/v1/documents/search"
+      :file_path: "/lighthouse/claims_api/benefits_documents/search/*"
```

Comment:
I'm not sure how these will work with the asterisk. It wouldn't know the lookup. To get the claimId I think you need something like the VA Notify service definition.
```      
:cache_multiple_responses:
        :uid_location: body
        :uid_locator: 'phone_number":"(.*?)"'
```
but obviously with our `data claimId` or whatever we have. Currently it finds the default but I don't think it will find any specific claim IDs this way. We can make this change at a later date though.